src/createContext.js

```diff
@@ -3,8 +3,8 @@ import loaders from './loaders';
 
 export default ({ pulseEngine }) => ({ request, connection }) => {
   if (request) {
-    const currentClients = clients(request.user);
-    const currentLoaders = loaders(currentClients, !!request.user);
+    const currentClients = clients(request.credentials);
+    const currentLoaders = loaders(currentClients, !!request.credentials);
```

Comment:
Can we use `Boolean(request.credentials)` instead of the double exclamation marks? I think it makes it more readable.